### PR TITLE
feature: adding script for bare metal execution of keycloak oid4vci

### PR DESCRIPTION
**Pull request by francis-pouatcha**

### CODE REVIEW

File: 0.start-kc-oid4vci.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# We work on following assumptions
+export DEV_DIR=~/dev
+export TOOLS_DIR=~/tools
+
+# change to you DEV_DIR and checkout keycloak
```
Comment (bengo237) [inline 0.start-kc-oid4vci.sh:7]: check_and_create_directories

File: 0.start-kc-oid4vci.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# checkout keycloak
+cd $DEV_DIR && git clone https://github.com/keycloak/keycloak.git
+
+# change into keycloak directory
```
Comment (bengo237) [inline 0.start-kc-oid4vci.sh:11]: Prevent network error with  low connexion

**Review by ArmandMeppa — APPROVED** [review]
Implemented some updates on the scripts to make it work

**Comment by bengo237** [comment]
Confirmed the script's functionality on an EC2 instance and reviewed [https://keycloak.solutions.adorsys.com/](https://keycloak.solutions.adorsys.com/) for testing purposes.

**Pull request merged**